src/data/nav/liveobjects.ts

```diff
@@ -106,6 +110,11 @@ export default {
           name: 'Swift plugin',
           external: true,
         },
+        {
+          link: 'https://sdk.ably.com/builds/ably/ably-java/main/javadoc/io/ably/lib/objects/RealtimeObjects.html',
+          name: 'Java plugin',
```

Comment:
I think this should match with JavaScript and be 'Java SDK' as it isn't technically a plugin like Swift.

Reply:
updated